xdsl/parser/attribute_parser.py

```diff
@@ -1124,6 +1125,17 @@ def try_parse_builtin_boolean_attr(
             return IntegerAttr(1 if value else 0, IntegerType(1))
         return None
 
+    def try_parse_builtin_none_type(self) -> NoneType | None:
```

Comment:
```suggestion
    def _parse_optional_none_type(self) -> NoneType | None:
```
Can you please add a call to this in `_parse_optional_builtin_type`?